tdrs-frontend/reports/zap.conf

```diff
@@ -0,0 +1,10 @@
+90034	IGNORE	(Cloud Metadata potentially exposed)
+90022	IGNORE	(Application Error Disclosure)
+40034	IGNORE	(.env information leak)
+10106	IGNORE	(HTTP Only Site)
```

Comment:
thanks for this in-line comment @riatzukiza regarding this one. I believe we need documentation on each of the findings determined to be false-positives as part of this PR, since we're ignoring them here. Its important for us to know why these are false-positives and would be a good idea to add this documentation to our docs directory. cc: @abottoms-coder

Reply:
As per our dicsussion during dev sync, we got rid of several of these after adding 404, and the remainder could go away in a production setting. We will circle back around to these when we have finished #1032